bolt-sidecar/src/config/signing.rs

```diff
@@ -13,4 +13,7 @@ pub struct SigningOpts {
     /// URL for the commit-boost sidecar
     #[clap(long, env = "BOLT_SIDECAR_COMMIT_BOOST_URL", conflicts_with("private_key"))]
```

Comment:
Let's rename env to `BOLT_SIDECAR_CB_SIGNER_URL` (to distinguish PBS module from commit boost)

And add `requires("commit_boost_jwt_hex")`